RELEASING.md

```diff
@@ -0,0 +1,61 @@
+# Releasing and Publishing of vodozemac
+
+While the release process can be handled manually, cargo-release has been
+configured to make it more convenient.
+
+By default, [`cargo-release`](https://github.com/crate-ci/cargo-release) assumes
+that no pull request is required to cut a release. However, since the vodozemac
+repo is set up so that each push requires a pull request, we need to slightly
+deviate from the default workflow.
+
+The procedure is as follows:
+
+1. Switch to a release branch:
+
+```bash
+git switch -c release-x.y.z
+```
+
+2. Prepare the release. This will update the README.md, CHANGELOG.md, and bump
+   the version.
+
+```bash
+cargo release --no-publish --no-tag --no-push --execute major|minor|patch|rc
+```
+
+3. Double-check and edit the changelog and README if necessary. Once you are
+   satisfied, push the branch and open a PR.
+
+```bash
+git push --set-upstream origin/release-x.y.z
+```
+
+4. Pass the review and merge the branch as you would with any other branch.
+
+5. Create a tag for your new release:
+
+```bash
+# Switch to main first.
+git switch main
+# Pull in the now-merged release commit(s).
+git pull
+# Run cargo-release to tag the commit, push the tag, and publish vodozemac on crates.io.
+cargo tag --execute
```

Comment:
Is something missing here? I don't know any `cargo tag` :eyes: and it seems the comment refers to instructions run afterwards.

Reply:
`cargo release tag --execute` duh...

Reply:
```suggestion
# Run cargo-release to tag the commit, push the tag, and publish vodozemac on crates.io.
cargo release tag --execute
```